packages/test/test-end-to-end-tests/src/test/messageSize.spec.ts

```diff
@@ -93,7 +93,11 @@ describeNoCompat("Message size", (getTestObjectProvider) => {
 
         const largeString = generateStringOfSize(maxMessageSizeInBytes + 1);
         const messageCount = 1;
-        setMapKeys(dataObject1map, messageCount, largeString);
+        try {
+            setMapKeys(dataObject1map, messageCount, largeString);
+            assert(false, "should throw");
```

Comment:
why does this fail? Also, would `assert.throws` work?

Reply:
I find it odd when we close container (on some error) but do not throw - that quite often causes other issues down the road. So, I prefer throwing error and stopping process that way. We use this pattern in some error flows, but do not use i other workflows. I want to see consistency here.